daisy/migrate.sh

```diff
@@ -0,0 +1,4 @@
+old=github.com/GoogleCloudPlatform/compute-image-tools/daisy
+new=github.com/GoogleCloudPlatform/compute-daisy
+grep -rl compute-image-tools/daisy . --exclude-dir=.git |
+  xargs sed -i "s|$old|$new|g"
```

Comment:
imo a 'find' invocation is safer and easier to understand, can give an example

that said if you want to do like this, the safe multiplatform inline sed is:
```suggestion
grep -rl compute-image-tools/daisy . --exclude-dir=.git |
  xargs sed -i"" "s|$old|$new|g"
```

Reply:
Ahh, cool! Changed to:

```bash
find . -path ./.git -prune -o -name "*.go" \
  -exec sed -i"" "s|$old|$new|g" {} +
```

Reply:
should be single quotes around `{}` : 
```
Only one instance  of  `{}'  is allowed within the command, and it must appear at the end, immediately before the `+'; it needs to be escaped (with a `\') or quoted to protect it from interpretation by the shell. 
```

Reply:
Done!